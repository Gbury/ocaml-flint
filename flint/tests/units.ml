let () =
  let f = Flint.FMPZ.of_int 42 in
  Format.printf "f:%a@." Flint.FMPZ.pp f;
  let z = Flint.FMPZ.to_z f in
  Format.printf "z:%a@." Z.pp_print z

let () =
  let f2 = Flint.FMPZ.of_z (Z.of_int 42) in
  Format.printf "z:%a@." Flint.FMPZ.pp f2
