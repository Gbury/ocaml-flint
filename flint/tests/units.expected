f:42
z:42
z:42
p:3*x^2+2*x+1
p:3*x^2+2*x+1
p:42
